sql/firefox_accounts_exact_mau28_by_dimensions_v1.sql

```diff
@@ -2,7 +2,11 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-derived-datasets.telemetry.firefox_accounts_exact_mau28_by_dimensions_v1`
 AS
 SELECT
-  * EXCEPT (generated_time, country),
+  * EXCEPT (generated_time, country)
+    -- We rename this column here to match the new standard of prefixing _mau
+    -- with the usage criterion; we can refactor to have the correct name in
+    -- the raw table the next time we need to make a change and backfill.
+    REPLACE (mau_tier1_inclusive AS seen_in_tier1_country_mau),
```

Comment:
does this work? I thought this required doing
```sql
* EXCEPT (generated_time, country, mau_tier1_inclusive),
mau_tier1_inclusive AS seen_in_tier1_country_mau
```

Reply:
You're right. Will change.